runner/ansible/roles/2.1.7/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '2.1.7'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    #  if no sid, then fail
+    sid=$(crm configure show | grep -m1 SID= | sed -e "s/.*SID=\(...\).*/\1/")
+    [[ $sid == "" ]] && exit 1
+    # if config file does not exist, exit fail
+    [[ -e /usr/sap/${sid}/SYS/global/hdb/custom/config/global.ini ]] || exit 1
+    # get ha_dr_provider_SAPHanaSR section from config file
+    ha_dr_provider_SAPHanaSR=$(sed -n -e '/ha_dr_provider_SAPHanaSR\]/,/\[/ p; /trace\]/,/\[/ p' /usr/sap/${sid}/SYS/global/hdb/custom/config/global.ini)
+    # if following not found, fail
+    [[ ${ha_dr_provider_SAPHanaSR} != "{{ expected[test_id + '.provider'] }}" ]] && exit 1
```

Comment:
All these 4 expected values are constants in fact. I think we should remove them from the variables files. The user shouldn't never touch them. As far as I know.
But well, in a second thought, let's keep in them where they are, maybe it is easier to find them

Reply:
Let's talk about this one. Not quite sure what you mean.

Reply:
I mean that the values on the expected dictionary `.provider`, `.path`, `.execution_order`, etc, are constants, so they never will change, I would vote to put them directly here rather than in the variables file